src/app/gateways/routes.ts

```diff
@@ -0,0 +1,82 @@
+import type { RouteRecordRaw } from 'vue-router'
+
+export const routes = () => {
+  const item = (): RouteRecordRaw[] => {
+    return [
+      {
+        path: 'gateways',
+        name: 'gateway-detail-view',
+        children: [
+          {
+            path: 'builtin/:gateway',
+            name: 'builtin-gateway-detail-tabs-view',
+            component: () => import('@/app/gateways/views/BuiltinGatewayDetailTabsView.vue'),
```

Comment:
**Comment**: We technically don’t need tabs here, yet, but I think there’s a good chance we will soon. In any case, we can always remove it if we don’t have any use for tabs here.